src/tools/compiletest/src/main.rs

```diff
@@ -15,6 +15,8 @@
 #![feature(test)]
 #![feature(libc)]
 
+#![cfg_attr(stage0, feature(question_mark))]
```

Comment:
https://github.com/rust-lang/rust/pull/37202 contains it too :D
The sooner it lands the better.